src/System.Net.Mail/src/System/Net/Mail/SmtpTransport.cs

```diff
@@ -205,11 +205,6 @@ internal void EndGetConnection(IAsyncResult result)
                 throw new ArgumentNullException(nameof(recipients));
             }
 
-            if (GlobalLog.IsEnabled && recipients.Count > 0)
```

Comment:
Why were these removed and how did the code work before (looks like this will always fail in the normal case)?